pkg/operator/controller/util.go

```diff
@@ -46,3 +50,155 @@ func mergeLabelsAndAnnotations(src, dest metav1.Object) {
 func deployClusterResources() bool {
 	return strings.ToLower(os.Getenv("DEPLOY_CLUSTER_RESOURCES")) != "false"
 }
+
+// MarkCrHealthyMessage marks the passed in CR as healthy. The CR object needs to be updated by the caller afterwards.
```

Comment:
I think this stuff belongs in cr.go because conditions are part of the cr.  Or make a conditions.go.

Reply:
Sure, don't care that much where it goes, if you prefer cr.go, that is where I will put it. Or a conditions.go doesn't matter much.